src/policy/fees.h

```diff
@@ -281,11 +281,12 @@ class FeeFilterRounder
     explicit FeeFilterRounder(const CFeeRate& minIncrementalFee);
 
     /** Quantize a minimum fee for privacy purpose before broadcast **/
```

Comment:
```suggestion
    /** Quantize a minimum fee for privacy purpose before broadcast . Not thread-safe due to use of FastRandomContext **/
```

Reply:
If this is not going to be changed, it's probably better to make it an explicit global.

Reply:
Oh, I am not suggestion to never change this. This can be changed when net_processing is multi-threaded. Until then documentation and sanitizers are good enough to leave the code as is for now.

Reply:
Isn't it as thread safe as a `class A { int a = 0;  public: int increment() { return ++a; } }`, which we'd solve by saying `static A my_a GUARDED_BY(cs_main);` ? And both could be made parallelisable by saying `static thread_local A my_a;` -- it's just that `GUARDED_BY` doesn't work with block-scope statics?

Just adding `// protected by cs_main` at the declaration of filterRounder, or moving it to be a module-level static and adding a `GUARDED_BY(cs_main)` seems better to me?

Reply:
[Updated](https://github.com/bitcoin/bitcoin/pull/19268#issuecomment-653873471).